examples/passport-login/src/server.ts

```diff
@@ -0,0 +1,78 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-passport-login
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {ApplicationConfig} from '@loopback/core';
+import express from 'express';
+import http from 'http';
+import {AddressInfo} from 'net';
+import pEvent from 'p-event';
+import {OAuth2LoginApplication} from './application';
+
+/**
+ * An express server with multiple apps
+ *
+ *  1. WEB App
+ *       a. An express web app which requires various user sign up provisions
+ *       b. The express router is mounted in the root '/' path
+ *  2. LB4 API server
+ *       a. LB4 application provides passport login services for the express app
+ *       b. The LB4 login apis are wrapped with session middleware to allow client sessions with user profiles
+ */
+export class ExpressServer {
```

Comment:
I am confused about the server+app structure here...so you setup an express server, and use the `webApp` and `lbApp` as two routers?